openshift/config/common/helm/babylon-config/templates/cross-cluster-backup.yaml

```diff
@@ -138,4 +167,102 @@ spec:
           - name: secret
             secret:
               secretName: cross-cluster-backup
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: restore
+  namespace: {{ .Values.crossClusterBackup.namespace }}
+  annotations:
+    openshift.io/description: >-
+      Service account used to restore backups.
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: cross-cluster-restore
+  namespace: {{ .Values.crossClusterBackup.namespace }}
+parameters:
+- description: Backup to Restore
+  displayName: Backup
+  name: BACKUP
+  required: true
+  value: manual
+- description: Image for restore Job
+  displayName: Image
+  name: IMAGE
+  required: true
+  value: quay.io/redhat-gpte/replik8s:latest
```

Comment:
`latest` ? do we want to pin version here ?

Reply:
good point, pinned version v0.3.1